pkg/vsphere/vm/vm_test.go

```diff
@@ -190,6 +191,13 @@ func TestVM(t *testing.T) {
 		t.Errorf("Failed to find fixed vm: %s", err)
 	}
 	assert.Equal(t, vm.Reference(), newVM.Reference())
+
+	// VM properties
+	var ovm mo.VirtualMachine
```

Comment:
This is exactly the sort of test where we should be using vcsim to return InvalidState - @dougm is there an easy way to do this fault injection?